source/_docs/configuration/basic.markdown

```diff
@@ -105,6 +106,15 @@ legacy_templates:
   required: false
   type: boolean
   default: false
+language:
+  description: "Default language used by Home Assistant. This may for example influence the language used by voice assistants. The language should be specified as an RFC 5646 language tag, and must be a language which Home Assistant is translated to."
+  required: false
+  type: string
+  default: `"en"`
+country:
+  description: "Country in which Home Assistant is running. This may for example influence radio settings to comply with local regulations. The country should be specified as an ISO 3166.1 alpha-2 code. Pick your country from the column **Code** of [Wikipedia's list of ISO 31661 alpha-2 officially assigned code codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)"
```

Comment:
```suggestion
  description: "Country in which Home Assistant is running. This may, for example, influence radio settings to comply with local regulations. The country should be specified as an ISO 3166.1 alpha-2 code. Pick your country from the column **Code** of [Wikipedia's list of ISO 31661 alpha-2 officially assigned code codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)"
```